### PR TITLE
Rebuild sitemap and others

### PR DESCRIPTION
**Pull request by montogeek**
I made a lot of mistakes when merging master into `rebuild` branch last week.
This PR rectifies those mistakes and also fix some TODOs along:
- Delete unused files.
- Update Alex configuration, https://github.com/webpack-contrib/npm-install-webpack-plugin/pull/124 was created to fix a warning.
- Include sitemap generation in build pipeline.
- Include CNAME generation in build pipeline.
- Removed unused dependencies.
- Add missing dependencies.
- Update SCSS import.

### CODE REVIEW

File: .alexignore [hunk 1]
```diff
@@ -1 +1 @@
-generated/plugins/*
+src/content/plugins/_*.md
```
Comment (jeremenichelli) [inline .alexignore]: Shouldn't we ignore more paths, like _loaders_? Or all the fetched files go to this one?

File: package.json [hunk 2]
```diff
@@ -25,19 +25,18 @@
...
     "clean": "rimraf ./dist",
-    "content": "node src/scripts/build-content-tree.js ./src/content ./src/_content.json",
     "prestart": "npm run clean",
     "start": "NODE_ENV=development webpack-dev-server --config webpack.dev.js --env.dev",
-    "prebuild": "npm run clean",
-    "build": "npm run content && npm run fetch && NODE_ENV=production webpack --config webpack.prod.js",
+    "content": "node src/scripts/build-content-tree.js ./src/content ./src/_content.json",
     "fetch": "sh src/scripts/fetch.sh",
+    "prebuild": "npm run clean",
+    "build": "npm run content && npm run fetch && NODE_ENV=production webpack --config webpack.prod.js && npm run sitemap && echo webpack.js.org > dist/CNAME",
```
Comment (jeremenichelli) [inline package.json]: Shouldn't _fetch_ run before content so the external markdown files are treated as content routes?

File: .alexignore (hunk 1 above)
Comment (montogeek) [reply .alexignore]: Yeah

File: package.json (hunk 2 above)
Comment (montogeek) [reply package.json]: Not sure, let me test

Comment (montogeek) [reply package.json]: Not, it is not the case. Routes are not created.

Comment (jeremenichelli) [reply package.json]: But, we don't need those routes? Or they aren't being created and that's a bug?

Comment (montogeek) [reply package.json]: No, we don't need them. I have compared each page to webpack.js.org and everything is there.

Comment (EugeneHlushko) [reply package.json]: I think the problem is, that `clean` doesnt remove fetched content, probably that is why you still get contents

Comment (EugeneHlushko) [reply package.json]: which should be the case for a clean build

Comment (montogeek) [reply package.json]: @EugeneHlushko I am not following you :D

Comment (EugeneHlushko) [reply package.json]: If you run `fetch` after `content` after pulling the repo, you wont get pages from fetch into directory tree

Comment (EugeneHlushko) [reply package.json]: content should be run after fetch

Comment (montogeek) [reply package.json]: @jeremenichelli @EugeneHlushko Absolutely right. Fixed in https://github.com/webpack/webpack.js.org/pull/2254/commits/fef8397c65a22dde49bfffbc6a97ccb73538aa4f

**Comment by montogeek** [comment]
Thank you @EugeneHlushko

**Pull request merged**